jar-filter/src/test/resources/abstract-function/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'org.jetbrains.kotlin.jvm' version '$kotlin_version'
-    id 'net.corda.plugins.jar-filter'
+    id 'net.corda.plugins.jar-filter' apply false
```

Comment:
Is not applied, because the plugin is needed to allow it's task to be imported only  e.g. `import net.corda.gradle.jarfilter.JarFilterTask` ?

Reply:
The plugin only needs to be on the plugin classpath so that the `Task` classes are visible. The plugin itself is implemented as:
```kotlin
class JarFilterPlugin : Plugin<Project> {
    override fun apply(project: Project) {
        project.logger.info("Applying JarFilter plugin")
    }
}
```
because it has nothing else to do :grin:.

Reply:
Indeed as explained in README.md.